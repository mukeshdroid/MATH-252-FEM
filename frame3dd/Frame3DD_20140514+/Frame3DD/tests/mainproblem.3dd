Input Data file for Verification-I - 2D structural frame analysis (N,m,Kg)

# node data ...
9				# number of nodes 
#.node  x       y       z       r                 units: meters

 1	0.0	0.0	0.0	0.0
 2	2.0	0.0	0.0	0.0
 3	4.0	0.0	0.0	0.0
 4	6.0	0.0	0.0	0.0
 5	8.0	0.0	0.0	0.0
 6	8.0	2.918	0.0	0.0
 7	6.0	2.184	0.0	0.0
 8	4.0	1.456	0.0	0.0	
 9	2.0	0.728	0.0	0.0
# reaction data ...
5				# number of nodes with reactions
#.n     x y z xx yy zz		1=fixed, 0=free

  1	1 1 1  1  1  0
  2	1 1 1  1  1  0
  3	1 1 1  1  1  0
  4	1 1 1  1  1  0
  5	1 1 1  1  1  0
  
# frame element data ...
15				# number of frame elements
#e n1 n2 Ax    Asy     Asz     Jxx     Iyy     Izz       E      G  roll density
#. .  .  in^2  in^2    in^2    in^4    in^4    in^4      ksi    ksi deg k/in^3/g

 1 1 2	1.0	1.0	1.0	1.0	1.0	0.01	1 11500  0  7.33e-7
 2 2 3	1.0	1.0	1.0	1.0	1.0	0.01	1 11500  0  7.33e-7
 3 3 4	1.0	1.0	1.0	1.0	1.0	0.01	1 11500  0  7.33e-7
 4 4 5  1.0	1.0	1.0	1.0	1.0	0.01	1 11500  0  7.33e-7
 5 5 6  1.0	1.0	1.0	1.0	1.0	0.01	1 11500  0  7.33e-7
 6 7 6  1.0	1.0	1.0	1.0	1.0	0.01	1 11500  0  7.33e-7
 7 8 7  1.0	1.0	1.0	1.0	1.0	0.01	1 11500  0  7.33e-7
 8 9 8  1.0	1.0	1.0	1.0	1.0	0.01	1 11500  0  7.33e-7
 9 1 9  1.0	1.0	1.0	1.0	1.0	0.01	1 11500  0  7.33e-7
 10 2 9  1.0	1.0	1.0	1.0	1.0	0.01	1 11500  0  7.33e-7
 11 2 8  1.0	1.0	1.0	1.0	1.0	0.01	1 11500  0  7.33e-7
 12 3 8  1.0	1.0	1.0	1.0	1.0	0.01	1 11500  0  7.33e-7
 13 4 8  1.0	1.0	1.0	1.0	1.0	0.01	1 11500  0  7.33e-7
 14 4 7 1.0	1.0	1.0	1.0	1.0	0.01	1 11500  0  7.33e-7
 15 4 6  1.0	1.0	1.0	1.0	1.0	0.01	1 11500  0  7.33e-7
 
0		# 1: include shear deformation
0		# 1: include geometric stiffness
5.0 		# exaggerate mesh deformations
1.0             # zoom scale for 3D plotting
-1.0		# x-axis increment for internal forces
		# if dx is -1 then internal force calculations are skipped.


2				# number of static load cases
				# Begin Static Load Case 1 of 2
		
# gravitational acceleration for self-weight loading (global)
#.gX		gY		gZ		
#.in./s^2	in./s^2		in./s^2
  0  		0		0

4				# number of loaded nodes
#.n     Fx       Fy     Fz      Mxx     Myy     Mzz
#.      kip      kip    kip     in.k    in.k    in.k
 6	200	-100	0.0	0.0	0.0	0.0
 7	0.0	-100	0.0	0.0	0.0	0.0
 8	0.0	-100	0.0	0.0	0.0	0.0
 9	0.0	-100	0.0	0.0	0.0	0.0


0				# number of uniform loads
0				# number of trapezoidal loads
0				# number of internal concentrated loads
0				# number of temperature loads

0				# number of nodes with prescribed displacements
#.n    Dx      Dy      Dz      Dxx     Dyy     Dzz
#.     in      in      in      rad.    rad.    rad.

				# End   Static Load Case 1 of 2

				# Begin Static Load Case 2 of 2

# gravitational acceleration for self-weight loading (global)
#.gX		gY		gZ		
#.in./s^2	in./s^2		in./s^2
   0		0		0

0				# number of loaded nodes
#.n     Fx       Fy     Fz      Mxx     Myy     Mzz
#       kip      kip    kip     in.k    in.k    in.k


0				# number of uniform loads
0				# number of trapezoidal loads
0				# number of internal concentrated loads

0			# number of temperature loads
#.e a      hy   hz   Ty+    Ty-    Tz+    Tz- 
#.  /degF  in   in   deg.F  deg.F  deg.F  deg.F

0				# number of nodes with prescribed displacements
#.n    Dx      Dy      Dz      Dxx     Dyy     Dzz
#.     in      in      in      rad     rad     rad


				# End   Static Load Case 2 of 2

0				# number of dynamic modes

# End of input data file for example A
