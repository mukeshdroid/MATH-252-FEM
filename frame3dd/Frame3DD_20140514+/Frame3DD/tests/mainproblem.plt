# FRAME3DD ANALYSIS RESULTS  http://frame3dd.sf.net/ VERSION 20140514+ 
# Input Data file for Verification-I - 2D structural frame analysis (N m Kg) 
# Tue Apr 19 14:19:30 2022
# G N U P L O T   S C R I P T   F I L E 
set autoscale
unset border
set pointsize 1.0
set xtics; set ytics; set ztics; 
unset zeroaxis
unset key
unset label
set size ratio -1    # 1:1 2D axis scaling 
# set view equal xyz # 1:1 3D axis scaling 
# NODE NUMBER LABELS
set label ' 1' at   0.0000e+00,   0.0000e+00,   0.0000e+00
set label ' 2' at   2.0000e+00,   0.0000e+00,   0.0000e+00
set label ' 3' at   4.0000e+00,   0.0000e+00,   0.0000e+00
set label ' 4' at   6.0000e+00,   0.0000e+00,   0.0000e+00
set label ' 5' at   8.0000e+00,   0.0000e+00,   0.0000e+00
set label ' 6' at   8.0000e+00,   2.9180e+00,   0.0000e+00
set label ' 7' at   6.0000e+00,   2.1840e+00,   0.0000e+00
set label ' 8' at   4.0000e+00,   1.4560e+00,   0.0000e+00
set label ' 9' at   2.0000e+00,   7.2800e-01,   0.0000e+00
# ELEMENT NUMBER LABELS
set label ' 1' at   1.0000e+00,   0.0000e+00,   0.0000e+00
set label ' 2' at   3.0000e+00,   0.0000e+00,   0.0000e+00
set label ' 3' at   5.0000e+00,   0.0000e+00,   0.0000e+00
set label ' 4' at   7.0000e+00,   0.0000e+00,   0.0000e+00
set label ' 5' at   8.0000e+00,   1.4590e+00,   0.0000e+00
set label ' 6' at   7.0000e+00,   2.5510e+00,   0.0000e+00
set label ' 7' at   5.0000e+00,   1.8200e+00,   0.0000e+00
set label ' 8' at   3.0000e+00,   1.0920e+00,   0.0000e+00
set label ' 9' at   1.0000e+00,   3.6400e-01,   0.0000e+00
set label ' 10' at   2.0000e+00,   3.6400e-01,   0.0000e+00
set label ' 11' at   3.0000e+00,   7.2800e-01,   0.0000e+00
set label ' 12' at   4.0000e+00,   7.2800e-01,   0.0000e+00
set label ' 13' at   5.0000e+00,   7.2800e-01,   0.0000e+00
set label ' 14' at   6.0000e+00,   1.0920e+00,   0.0000e+00
set label ' 15' at   7.0000e+00,   1.4590e+00,   0.0000e+00
# set parametric
# set view 60, 70,  1.00 
# set view equal xyz # 1:1 3D axis scaling 
# unset key
# set xlabel 'x'
# set ylabel 'y'
# set zlabel 'z'
set title "Input Data file for Verification-I - 2D structural frame analysis (N m Kg) \nanalysis file: mainproblem.3dd   deflection exaggeration: 5.0   load case 1 of 2 "
unset clip; 
set clip one; set clip two
set xyplane 0 
  plot '/tmp/mainproblem-msh' u 2:3 t 'undeformed mesh' w lp lw 1 lt 5 pt 6, '/tmp/mainproblem-mshf.001' u 1:2 t 'load case 1 of 2' w l lw 1 lt 3
# splot '/tmp/mainproblem-msh' u 2:3:4 t 'load case 1 of 2' w lp  lw 1 lt 5 pt 6, '/tmp/mainproblem-mshf.001' u 1:2:3 t 'load case 1 of 2' w l lw 1 lt 3
pause -1
set title "Input Data file for Verification-I - 2D structural frame analysis (N m Kg) \nanalysis file: mainproblem.3dd   deflection exaggeration: 5.0   load case 2 of 2 "
unset clip; 
set clip one; set clip two
set xyplane 0 
  plot '/tmp/mainproblem-msh' u 2:3 t 'undeformed mesh' w lp lw 1 lt 5 pt 6, '/tmp/mainproblem-mshf.002' u 1:2 t 'load case 2 of 2' w l lw 1 lt 3
# splot '/tmp/mainproblem-msh' u 2:3:4 t 'load case 2 of 2' w lp  lw 1 lt 5 pt 6, '/tmp/mainproblem-mshf.002' u 1:2:3 t 'load case 2 of 2' w l lw 1 lt 3
