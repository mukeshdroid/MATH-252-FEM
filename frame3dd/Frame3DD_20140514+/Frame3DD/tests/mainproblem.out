________________________________________________________________________________
Frame3DD version: 20140514+               http://frame3dd.sf.net/
GPL Copyright (C) 1992-2015, Henri P. Gavin 
Frame3DD is distributed in the hope that it will be useful but with no warranty.
For details see the GNU Public Licence: http://www.fsf.org/copyleft/gpl.html
________________________________________________________________________________

Input Data file for Verification-I - 2D structural frame analysis (N m Kg) 
Mon Apr 18 12:46:21 2022
________________________________________________________________________________
In 2D problems the Y-axis is vertical.  In 3D problems the Z-axis is vertical.
________________________________________________________________________________
    9 NODES              5 FIXED NODES       15 FRAME ELEMENTS   2 LOAD CASES   
________________________________________________________________________________
N O D E   D A T A                                           R E S T R A I N T S
  Node       X              Y              Z         radius  Fx Fy Fz Mx My Mz
    1       0.000000       0.000000       0.000000    0.000   1  1  1  1  1  0
    2       2.000000       0.000000       0.000000    0.000   1  1  1  1  1  0
    3       4.000000       0.000000       0.000000    0.000   1  1  1  1  1  0
    4       6.000000       0.000000       0.000000    0.000   1  1  1  1  1  0
    5       8.000000       0.000000       0.000000    0.000   1  1  1  1  1  0
    6       8.000000       2.918000       0.000000    0.000   0  0  0  0  0  0
    7       6.000000       2.184000       0.000000    0.000   0  0  0  0  0  0
    8       4.000000       1.456000       0.000000    0.000   0  0  0  0  0  0
    9       2.000000       0.728000       0.000000    0.000   0  0  0  0  0  0
F R A M E   E L E M E N T   D A T A					(local)
  Elmnt  J1    J2     Ax   Asy   Asz    Jxx     Iyy     Izz       E       G roll  density
    1     1     2    1.0   1.0   1.0    1.0     1.0     0.0      1.0 11500.0   0 7.33e-07
    2     2     3    1.0   1.0   1.0    1.0     1.0     0.0      1.0 11500.0   0 7.33e-07
    3     3     4    1.0   1.0   1.0    1.0     1.0     0.0      1.0 11500.0   0 7.33e-07
    4     4     5    1.0   1.0   1.0    1.0     1.0     0.0      1.0 11500.0   0 7.33e-07
    5     5     6    1.0   1.0   1.0    1.0     1.0     0.0      1.0 11500.0   0 7.33e-07
    6     7     6    1.0   1.0   1.0    1.0     1.0     0.0      1.0 11500.0   0 7.33e-07
    7     8     7    1.0   1.0   1.0    1.0     1.0     0.0      1.0 11500.0   0 7.33e-07
    8     9     8    1.0   1.0   1.0    1.0     1.0     0.0      1.0 11500.0   0 7.33e-07
    9     1     9    1.0   1.0   1.0    1.0     1.0     0.0      1.0 11500.0   0 7.33e-07
   10     2     9    1.0   1.0   1.0    1.0     1.0     0.0      1.0 11500.0   0 7.33e-07
   11     2     8    1.0   1.0   1.0    1.0     1.0     0.0      1.0 11500.0   0 7.33e-07
   12     3     8    1.0   1.0   1.0    1.0     1.0     0.0      1.0 11500.0   0 7.33e-07
   13     4     8    1.0   1.0   1.0    1.0     1.0     0.0      1.0 11500.0   0 7.33e-07
   14     4     7    1.0   1.0   1.0    1.0     1.0     0.0      1.0 11500.0   0 7.33e-07
   15     4     6    1.0   1.0   1.0    1.0     1.0     0.0      1.0 11500.0   0 7.33e-07
  Neglect shear deformations.
  Neglect geometric stiffness.

L O A D   C A S E   1   O F   2  ... 

   Gravity X =  0.0    Gravity Y =  0.0    Gravity Z =  0.0 
   4 concentrated loads
   0 uniformly distributed loads
   0 trapezoidally distributed loads
   0 concentrated point loads
   0 temperature loads
   0 prescribed displacements
 N O D A L   L O A D S  +  E Q U I V A L E N T   N O D A L   L O A D S  (global)
  Node        Fx          Fy          Fz          Mxx         Myy         Mzz
     6     200.000    -100.000       0.000       0.000       0.000       0.000
     7       0.000    -100.000       0.000       0.000       0.000       0.000
     8       0.000    -100.000       0.000       0.000       0.000       0.000
     9       0.000    -100.000       0.000       0.000       0.000       0.000

L O A D   C A S E   2   O F   2  ... 

   Gravity X =  0.0    Gravity Y =  0.0    Gravity Z =  0.0 
   0 concentrated loads
   0 uniformly distributed loads
   0 trapezoidally distributed loads
   0 concentrated point loads
   0 temperature loads
   0 prescribed displacements

E L A S T I C   S T I F F N E S S   A N A L Y S I S   via  L D L'  decomposition


L O A D   C A S E   1   O F   2  ... 

N O D E   D I S P L A C E M E N T S  					(global)
  Node    X-dsp       Y-dsp       Z-dsp       X-rot       Y-rot       Z-rot
     1    0.0         0.0         0.0         0.0         0.0        17.196104
     2    0.0         0.0         0.0         0.0         0.0       -88.696933
     3    0.0         0.0         0.0         0.0         0.0       -47.872157
     4    0.0         0.0         0.0         0.0         0.0       -98.246961
     5    0.0         0.0         0.0         0.0         0.0      -139.074877
     6 1215.520980 -603.649343    0.0         0.0         0.0      -422.366309
     7  666.504758 -211.534409    0.0         0.0         0.0      -252.541315
     8  216.352748  -59.822723    0.0         0.0         0.0       -74.538573
     9  108.932533  -68.431346    0.0         0.0         0.0      -119.758290
F R A M E   E L E M E N T   E N D   F O R C E S				(local)
  Elmnt  Node       Nx          Vy         Vz        Txx        Myy        Mzz
     1      1      0.0        -1.073      0.0        0.0        0.0       -0.543
     1      2      0.0         1.073      0.0        0.0        0.0       -1.602
     2      2      0.0        -2.049      0.0        0.0        0.0       -2.253
     2      3      0.0         2.049      0.0        0.0        0.0       -1.844
     3      3      0.0        -2.192      0.0        0.0        0.0       -1.940
     3      4      0.0         2.192      0.0        0.0        0.0       -2.444
     4      4      0.0        -3.560      0.0        0.0        0.0       -3.356
     4      5      0.0         3.560      0.0        0.0        0.0       -3.764
     5      5    206.871c      1.914      0.0        0.0        0.0        3.764
     5      6   -206.871c     -1.914      0.0        0.0        0.0        1.822
     6      7   -178.511t     -2.006      0.0        0.0        0.0       -1.340
     6      6    178.511t      2.006      0.0        0.0        0.0       -2.934
     7      8   -174.362t     -0.641      0.0        0.0        0.0        0.154
     7      7    174.362t      0.641      0.0        0.0        0.0       -1.519
     8      9    -48.810t     -2.217      0.0        0.0        0.0       -2.572
     8      8     48.810t      2.217      0.0        0.0        0.0       -2.147
     9      1    -37.097t     -0.094      0.0        0.0        0.0        0.543
     9      9     37.097t      0.094      0.0        0.0        0.0       -0.744
    10      2     93.999c     10.281      0.0        0.0        0.0        4.169
    10      9    -93.999c    -10.281      0.0        0.0        0.0        3.316
    11      2    -56.472t     -0.208      0.0        0.0        0.0       -0.314
    11      8     56.472t      0.208      0.0        0.0        0.0       -0.200
    12      3     41.087c      4.947      0.0        0.0        0.0        3.784
    12      8    -41.087c     -4.947      0.0        0.0        0.0        3.418
    13      4     84.937c     -1.068      0.0        0.0        0.0       -1.417
    13      8    -84.937c      1.068      0.0        0.0        0.0       -1.225
    14      4     96.856c      3.265      0.0        0.0        0.0        4.272
    14      7    -96.856c     -3.265      0.0        0.0        0.0        2.859
    15      4    -53.504t      1.147      0.0        0.0        0.0        2.944
    15      6     53.504t     -1.147      0.0        0.0        0.0        1.112
R E A C T I O N S							(global)
  Node        Fx          Fy          Fz         Mxx         Myy         Mzz
     1     -34.827     -13.850       0.000       0.000       0.000       0.0  
     2     -55.813      59.618       0.000       0.000       0.000       0.0  
     3      -4.947      40.944       0.000       0.000       0.000       0.0  
     4    -102.499     102.857       0.000       0.000       0.000       0.0  
     5      -1.914     210.431       0.000       0.000       0.000       0.0  
R M S    R E L A T I V E    E Q U I L I B R I U M    E R R O R: 3.277e-14

L O A D   C A S E   2   O F   2  ... 

N O D E   D I S P L A C E M E N T S  					(global)
  Node    X-dsp       Y-dsp       Z-dsp       X-rot       Y-rot       Z-rot
F R A M E   E L E M E N T   E N D   F O R C E S				(local)
  Elmnt  Node       Nx          Vy         Vz        Txx        Myy        Mzz
     1      1      0.0         0.0        0.0        0.0        0.0        0.0  
     1      2      0.0         0.0        0.0        0.0        0.0        0.0  
     2      2      0.0         0.0        0.0        0.0        0.0        0.0  
     2      3      0.0         0.0        0.0        0.0        0.0        0.0  
     3      3      0.0         0.0        0.0        0.0        0.0        0.0  
     3      4      0.0         0.0        0.0        0.0        0.0        0.0  
     4      4      0.0         0.0        0.0        0.0        0.0        0.0  
     4      5      0.0         0.0        0.0        0.0        0.0        0.0  
     5      5      0.0         0.0        0.0        0.0        0.0        0.0  
     5      6      0.0         0.0        0.0        0.0        0.0        0.0  
     6      7      0.0         0.0        0.0        0.0        0.0        0.0  
     6      6      0.0         0.0        0.0        0.0        0.0        0.0  
     7      8      0.0         0.0        0.0        0.0        0.0        0.0  
     7      7      0.0         0.0        0.0        0.0        0.0        0.0  
     8      9      0.0         0.0        0.0        0.0        0.0        0.0  
     8      8      0.0         0.0        0.0        0.0        0.0        0.0  
     9      1      0.0         0.0        0.0        0.0        0.0        0.0  
     9      9      0.0         0.0        0.0        0.0        0.0        0.0  
    10      2      0.0         0.0        0.0        0.0        0.0        0.0  
    10      9      0.0         0.0        0.0        0.0        0.0        0.0  
    11      2      0.0         0.0        0.0        0.0        0.0        0.0  
    11      8      0.0         0.0        0.0        0.0        0.0        0.0  
    12      3      0.0         0.0        0.0        0.0        0.0        0.0  
    12      8      0.0         0.0        0.0        0.0        0.0        0.0  
    13      4      0.0         0.0        0.0        0.0        0.0        0.0  
    13      8      0.0         0.0        0.0        0.0        0.0        0.0  
    14      4      0.0         0.0        0.0        0.0        0.0        0.0  
    14      7      0.0         0.0        0.0        0.0        0.0        0.0  
    15      4      0.0         0.0        0.0        0.0        0.0        0.0  
    15      6      0.0         0.0        0.0        0.0        0.0        0.0  
R E A C T I O N S							(global)
  Node        Fx          Fy          Fz         Mxx         Myy         Mzz
     1       0.000       0.000       0.000       0.000       0.000       0.0  
     2       0.000       0.000       0.000       0.000       0.000       0.0  
     3       0.000       0.000       0.000       0.000       0.000       0.0  
     4       0.000       0.000       0.000       0.000       0.000       0.0  
     5       0.000       0.000       0.000       0.000       0.000       0.0  
R M S    R E L A T I V E    E Q U I L I B R I U M    E R R O R: 3.277e-14

